configure

```diff
@@ -2851,12 +2851,18 @@ fi
 
 
 #!/bin/sh
-if test -d ./src/boost.tar.gz;
+if test "$(uname -m)" == "x86_64";
   then
-      echo 'found boost header sources and tar archive;\n using what is there.'
-  else
-      echo "untarring boost include tree...";
-      tar -zxf ./src/boost.tar.gz -C ./src;
+    CPPFLAGS="-I. ${CPPFLAGS}"
```

Comment:
Is this change needed? I think we should leave it in `Makevars.in`. It shouldn't make a difference because there will be no headers anyway in `-I.`.

Reply:
`-I.` is in the context of `src/` folder and there we have [handlesegfault.h](https://github.com/ahmohamed/NetPathMiner/blob/master/src/handlesegfault.h) and few more.
By removing the `-I.` from Makevars.in and moving it here we add `-I.` only when `src/boost` is used.
If `-I.` is always in Makevars.in then one can do `#include <handlesegfault.h>` by mistake.

Reply:
Actually, don't we need those headers to be included? `init.h` and `hme3m.h` are referenced in cpp files.
https://github.com/ahmohamed/NetPathMiner/blob/master/src/PathRanker.cpp#L40

Reply:
Right! They need to be included!
Currently they are included with `#include "relative/path.h"`.  Note the usage of `"`.
If `-I.` is added then one can mistakenly include them with: `#include <hme3m.h>`. Note the usage of `<` and `>`, i.e. it may load `hme3m.h` from some other location on the INCLUDE_PATH.

Reply:
I'll revert it!
I see that this change causes confusion. 
It is an improvement for a hypothetical use case anyway.

Reply:
Sorry I just noticed the `<>`. I'll resolve this and merge. I'll handle `configure.ac` at a later stage once we can confirm it's building smoothly within the BBS.